index.js

```diff
@@ -1,16 +1,22 @@
 const Id = require('hypercore-id-encoding')
 const Hyperbee = require('hyperbee')
 const ReadyResource = require('ready-resource')
+const b4a = require('b4a')
 const { keyEncoding, valueEncoding } = require('./encoding.js')
```

Comment:
Detail, but I'd consider `const { keyEncoding: contentKeyEncoding}` to avoid mistakes in the future (all too easy to accidentally pass in `keyEncoding` as keyEncoding, instead of `metadata` or `content`. Or easier: putting all encoding logic in the './encodings.js' file, so you an import the metadata- and content encodings from there, then there's less potential for confusion in this file